deps/ccommon/include/cc_itt.h

```diff
@@ -0,0 +1,71 @@
+#pragma once
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <cc_define.h>
+
+#ifdef CC_ITT
+#include "ittnotify.h"
```

Comment:
all includes must be above `extern "C"`. You are changing linkage of not your functions.

Reply:
Done.